packages/ra-ui-materialui/src/field/EmailField.tsx

```diff
@@ -34,9 +34,15 @@ const EmailField = memo<EmailFieldProps>((props: EmailFieldProps) => {
             className={className}
             href={`mailto:${value}`}
             onClick={stopPropagation}
-            {...sanitizeFieldRestProps(rest)}
         >
-            {value}
+            <Typography
+                component="span"
+                variant="body2"
+                className={className}
+                {...sanitizeFieldRestProps(rest)}
+            >
+                {value}
+            </Typography>
```

Comment:
I think you can pass these props directly to the `Link` component. See the example above this link: https://material-ui.com/components/links/#api

Reply:
Why should I? The props have to end up in the Typography element anyway.

Reply:
nevermind, you're right, I'll change that.